site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -356,7 +356,9 @@
 
 {% if core.getUser().accessFullGrading() %}
     {% include 'submission/homework/PreviousSubmissionForm.twig' %}
-    {% include 'submission/homework/BulkUploadProgressBox.twig' %}
+    {% not is_vcs and not is_notebook %}
```

Comment:
I think you forgot the `if` here, twig is complaining this isn't a block when you visit a gradeable view grade page

```

Server Error
Oh no! Something irrecoverable has happened...

app\exceptions\OutputException (Code: 0) thrown in /usr/local/submitty/site/app/libraries/Output.php (Line 371) by:
            throw new OutputException("{$e->getMessage()} in {$e->getFile()}:{$e->getLine()}");


Message:
A block must start with a tag name. in /usr/local/submitty/site/app/templates/submission/homework/SubmitBox.twig:359

Stack Trace:
#0 /usr/local/submitty/site/app/views/submission/HomeworkView.php(457): app\libraries\Output->renderTwigTemplate()
#1 /usr/local/submitty/site/app/views/submission/HomeworkView.php(64): app\views\submission\HomeworkView->renderSubmitBox()
#2 unknown file(unknown line): app\views\submission\HomeworkView->showGradeable()
#3 /usr/local/submitty/site/app/libraries/Output.php(233): call_user_func_array()
#4 /usr/local/submitty/site/app/libraries/Output.php(202): app\libraries\Output->renderTemplate()
#5 /usr/local/submitty/site/app/controllers/student/SubmissionController.php(194): app\libraries\Output->renderOutput()
#6 unknown file(unknown line): app\controllers\student\SubmissionController->showHomeworkPage()
#7 /usr/local/submitty/site/app/libraries/routers/WebRouter.php(254): call_user_func_array()
#8 /usr/local/submitty/site/app/libraries/routers/WebRouter.php(232): app\libraries\routers\WebRouter->run()
#9 /usr/local/submitty/site/public/index.php(131): app\libraries\routers\WebRouter::getWebResponse()

                      ,--.    ,--.
                     (( O))--(( O))
                   ,'_`--'____`--'_`.
                  _:  ____________  :_      _____
                 | | ||::::::::::|| | |        \ \
                 | | ||::::::::::|| | |         \
                 | | ||::::::::::|| | |
                 |_| |/__________\| |_|
                   |________________|
                __..-'            `-..__
             .-| : .----------------. : |-.
           ,\ || | |\______________/| | || /.
          /`.\:| | ||  __  __  __  || | |;/,'\
         :`-._\;.| || '--''--''--' || |,:/_.-':
         |    :  | || .----------. || |  :    |
         |    |  | || '----SSt---' || |  |    |
         |    |  | ||   _   _   _  || |  |    |
         :,--.;  | ||  (_) (_) (_) || |  :,--.;
         (`-'|)  | ||______________|| |  (|`-')
          `--'   | |/______________\| |   `--'
                 |____________________|
                  `.________________,'
                   (_______)(_______)
                   (_______)(_______)
                   (_______)(_______)
                   (_______)(_______)
                  |        ||        |
                  '--------''--------'
    

````

Reply:
Sorry forgot to push the the latest commit